configure.ac

```diff
@@ -85,12 +85,17 @@ AS_IF([test x"$with_mpi" = x"yes"],
 AM_CONDITIONAL([WITH_MPI], [test x"$build_mpi" = x"yes"])
 AM_CONDITIONAL([WITH_MPI_TESTS], [test x"$build_mpi" = x"yes" -a -z "$SKIP_MPI_TEST"])
 
+#Build with OpenACC suport.  Default is 'no'
+AC_ARG_ENABLE([acc],
+  [AS_HELP_STRING([--enable-acc], [Build with OpenACC support.  Defaults to 'no'])] )
+AS_IF([test ${enable_acc:no} = "yes"], [enable_acc=yes], [enable_acc=no])
+AM_CONDITIONAL([ENABLE_ACC], [test $enable_acc = "yes"])
+
```

Comment:
When enable-acc is set to yes (true), is there a second test to ensure you are using nvc/nvfortran and an appropriate error message?  If not, is the plan to simply fail the check_headers for `openacc.h`?

Reply:
I haven't even thought of that, putting in a second check is needed here since users could be using gcc which openacc.h exists but -acc is the wrong flag. @rem1776 will your PR have this fix?  If not, I'll put in the fix here :)

Reply:
Ignore above comment, -acc flag has nothing to do with enable-acc here, but since we're only supporting nvhpc compilers for fregird_acc so far, that check should be put in.